charts/clamav/templates/clamav-certificate.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.clamav.certificate.create }}
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
Please specify `cert-manager.io/v1alpha2` cert-manager.io instead of `certmanager.k8s.io/v1alpha1`.
The new Certificate
[Migration from Jetstack cert-manager v0.8 resources to v0.13.1](https://github.com/UKHomeOffice/application-container-platform/blob/master/docs/how-to-docs/cert-manager-upgrade-from-v0.8.md) has more information on that subject